app/assets/stylesheets/application.css.scss

```diff
@@ -255,8 +278,24 @@ main {
     top: 9px;
     right: 2px;
     position: absolute;
+    border-radius: 5px;
+    color: white;
+    background-color: black;
+    border: none;
+}
+
+.avatar-selection{
+font-size: 13px
```

Comment:
Line should be indented 2 spaces, but was indented 0 spaces<br>Declaration should be terminated by a semicolon